doc/tutorial.rst

```diff
@@ -92,7 +92,7 @@ but doesn't want to wait for a reply, the
 Note that the :func:`reply_cb` and :func:`inform_cb` callback functions are both
```

Comment:
Strange foible of sphinx, if you do the :func: thing naked it adds the parens for you, but if you do it as :func:`reply_cb <katcp.blah.reply_cb>` it takes your description literally and doesn't add parens. So in this case it should be fine without the ().